Add deployment info to README (#18)
anz-bank/sysl-website: README.md

### Deployment

> A development version is currently deployed using Netlify. Deployment previews are enabled ,so each PR should have a unique deployment preview link which can be found in the Github Status Checks.

Reviewer: ```suggestion
A development version i deployed using Netlify. Deployment previews are enabled, so each PR has a unique deployment preview link which can be found in the Github Status Checks.
```
